### DISCUSSION TITLE
On joblib file size

### DISCUSSION THREAD

**Discussion by mh-tanver** [AstroBridge/BDLensing#218] [Q&A] [answered]
Original joblib file size was 40 mb. I compressed it to 27mb but it's still too big. I'm seeking suggestions on further compressions and anything related with file size. Here is the file drive link,
Original file name: DESIJ1537+1337_mcmc_output.joblib.
Compressed file name: modified_data.joblib.

https://drive.google.com/drive/folders/1VLvXsgVwnoT6xDJWp0RzRkyjLdP1T87j?usp=drive_link

**Comment by mamun-nahid** [comment]
In my case, the file was huge because of the `initial_samples` dictionary and removing it reduced the file size significantly. But in your file `initial_samples ` is set to `none`, so there's no problem there. 

You can check running MCMC again with `n_run = 10` and then save the output file again. My joblib file was way too large when I used `n_run =500` but when I used `n_run = 10` in the last run and saved the file again, the file size became less than 200kb. Although I can't find any reason for that to happen. Can you check it?

**Reply by mh-tanver** [reply]
Alright, let me try that. Thanks for the suggestion.

**Reply by mh-tanver** [reply, marked as answer]
Did you run the cornerplots for n = 10? or just the traceplots and saved the file?
